Adjust CHANGES to match recent patch releases (#1406)
mirage/irmin: CHANGES.md

  - use the pre_hash function to compute entry keys instead of
    their raw binary representation (#1308, @samoht)

### Added

- **irmin**

>    - Add `Store.Private.Node.Val.length`. (#1315, @Ngoguey42)
>    - Add `Store.Tree.length`. (#1316, @Ngoguey42)

Reviewer: ```suggestion
   - Added `Store.Private.Node.Val.length`. (#1315, @Ngoguey42)
   - Added `Store.Tree.length`. (#1316, @Ngoguey42)
```

    (#1377, @samoht)
  - Moved `Irmin_pack.Store.Atomic_write` into its own module (#1378, @samoht)

## 2.5.3 (2021-04-13)

### Fixed

- **irmin**
  - Fixed a bug causing equality functions derived from `Store.tree_t` to return
    false-negatives. (#1371, @CraigFe)

### Added

- **irmin**

>   - Add `Store.Tree.is_empty`. (#1373, @CraigFe)

Reviewer: ```suggestion
  - Added `Store.Tree.is_empty`. (#1373, @CraigFe)
```

## 2.5.2 (2021-04-08)

### Fixed

- **irmin**
  - The `Tree.update_tree` and `Tree.add_tree` functions now interpret adding
    an empty subtree as a remove operation, rather than adding an empty
    directory.  (#1335, @craigfe)

- **irmin-pack**

>   - Fix a performance regression where all caches where always cleaned by

Reviewer: ```suggestion
  - Fixed a performance regression where all caches were always cleaned by
```
